### ISSUE TITLE
Debug negative volumes from controller run

### ISSUE THREAD

**Issue by psakievich** [psakievich/hfm-fsi#5]

**Comment by kevmoor** [comment]
Perhaps a dumb question, or perhaps you've already looked into it, but is there a constraint in the mesh deformation to prevent cells from getting too small?  My potential worry is that even by removing the root areas where the strain is largest, the mesh may still become very skewed, though not negative, and cause numerical issues. If so, perhaps for this sprint, just checking that it isn't causing any issues for the design load cases we run and note when it might become an issue for future work.

**Comment by ashesh2512** [comment]
No, there isn't an explicit constraint in the mesh deformation. If constraints were to be introduced, what happens if they are violated?Does the mesh need to be re-meshed by the CFD solver?

**Comment by kevmoor** [comment]
Without being super familiar with the algorithm and interface, I'm guessing the underlying beam model in openFAST is at a much coarser discretization than the CFD mesh, so the finer mesh just needs to match the coarser local curvatures and strains?  Just guessing here at how the deformation works, but if it say adds and reduces lengths based on the overall required deformation, we could add the min constraint, if it is activated, then the affected cells are locked at the minimum size and the remaining deformation needed would be calculated and rerun through the deformation algorithm again, and if it exceeds a certain number of iterations, then it errors. It should act kind of like a slinky when you hold both ends vertical - the center top will "min" out but not go negative to meet the overall 180 degree curvature requirement.

**Comment by pscrozi** [comment]
We wanted to see where we were getting negative volumes. Phil and Ashesh discussed. It ran, but we didn't get to viz. Still need to viz.

**Comment by psakievich** [comment]
The wrong defaults were/are being applied to the ramping disablers such that all meshes were getting fully deformed.  This can be fixed in the input deck, and https://github.com/Exawind/nalu-wind/pull/1216 restores the default behavior.

**Comment by pscrozi** [comment]
Wrong default has been removed and merged to master. Negative volumes will now go away. Phil working on unified mesh on Frontier.
